std-bits/base/src/main/java/org/enso/base/Array_Utils.java

```diff
@@ -0,0 +1,7 @@
+package org.enso.base;
+
+public class Array_Utils {
+  public static byte[] ensureByteArray(byte[] input) {
```

Comment:
May be worth to add a comment to this method, especially as it indeed may look peculiar (since it is just identity), so a short explanation of its purpose would help.